### DISCUSSION TITLE
Dual Wallet support on one main node

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#5663] [Ideas]
Due to various reasons, it is very desirable to have two wallets on the main node to be synced at a same time. One wallet tied to the plots and farming as usual. One wallet would be mainly used to store accumulated XCH, while the other wallet can be used to communicate with say an exchange to test it out before potential bug or network issues that can result in lost rewards. A 2nd wallet can be setup easily on the main node. But the GUI would only be able to sync one.  Would need instructions on how to sync both wallet at a same time.

**Comment by pausto** [comment]
I join the problem. There are 2 wallets in the GUI. One is used constantly, therefore it is synchronized. I enter the second sometimes. He does not see the transactions made. Add a button to FORCE sync, from the first wallet.
